setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 
 setup( name='agilepy',
-       version='1.6.2',
+       version='1.6.3',
        author='Baroncelli Leonardo, Addis Antonio, Bulgarelli Andrea, Parmiggiani Nicolò, Ambra Di Piano, Gabriele Panebianc',
```

Comment:
A 'o' is missing in "Gabriele Panebianc"

Reply:
Thanks, fixed in the latest commit